src/main.rs

```diff
@@ -1150,6 +1166,51 @@ mod tests {
                 3
             );
 
+            let nics = guest.get_net_interfaces()?;
+            let nic_eth0 = "eth0";
+            let nic_lo = "lo";
+            let nic_f = nics.split('\n').filter(|s| s != &nic_eth0 && s != &nic_lo);
+            let nic_v: Vec<&str> = nic_f.collect();
+
+            let ip = guest.network.guest_ip.clone();
+            let ip_v: Vec<i32> = ip.split('.').map(|s| s.parse().unwrap()).collect();
+            let dest_ip = format!("{}.{}.{}.{}", ip_v[0], ip_v[1], ip_v[2], ip_v[3] + 5);
+            let src_ip = format!("{}.{}.{}.{}", ip_v[0], ip_v[1], ip_v[2], ip_v[3] + 6);
```

Comment:
I'd rather you create a new std::net::Ipv4Addr and then formatted it to a string with  {}

Reply:
create ip like this to avoid ip conflict with other vms.

Reply:
I want you to go via a std::net::Ipv4Addr rather than printing the octets yourself. Please read the comments carefully.

Reply:
Oh, sorry, got it. :-P I will change it.